virtualizarr/tests/test_integration.py

```diff
@@ -0,0 +1,18 @@
+from virtualizarr import open_virtual_dataset
+
+from pprint import pprint
+
+
+def test_numpy_arrays_to_inlined_kerchunk_refs(netcdf4_file):
+    from kerchunk.hdf import SingleHdf5ToZarr
+
+    # test inlining all the variables
+    expected = SingleHdf5ToZarr(netcdf4_file, spec=1, inline_threshold=1e9).translate()
```

Comment:
IIUC this should inline all the chunks smaller than 1GB (i.e. all of them). But kerchunk doesn't seem to be doing this. Any ideas what I'm missing @norlandrhagen ?

Reply:
See also https://github.com/fsspec/kerchunk/issues/445

Reply:
Resolved by pinning to `kerchunk=0.2.2`, now I can see inlined data.